awx/main/management/commands/provision_instance.py

```diff
@@ -35,7 +35,7 @@ def _register_hostname(self, hostname, node_type, uuid):
 
             from awx.main.management.commands.register_queue import RegisterQueue
 
-            (changed, instance) = Instance.objects.register(ip_address=os.environ.get('MY_POD_IP'), node_type='control', uuid=settings.SYSTEM_UUID)
+            (changed, instance) = Instance.objects.register(ip_address=os.environ.get('MY_POD_IP'), node_type='control', _uuid=settings.SYSTEM_UUID)
```

Comment:
The underscore here looks weird... can we revert that part?

Reply:
Fixed